vanilla-framework/vanilla-framework-react
# 50: Add List component

# Done
- Added [List](https://docs.vanillaframework.io/en/patterns/lists) component and ListItem sub-component
- Includes Default, Ticked, Divided, Inline, Middot and Split options
- Added stories to Storybook
- Added snapshot tests
- **Note:** I started building it out so that 'stepped' and 'stepped-detailed' could also be modifiers, but I think it would work better as a separate component as it's wired a bit differently to the rest of the list types

# QA
- Pull code
- Run `yarn clean` and `yarn serve` and navigate to http://localhost:6006/
- Run `yarn lint` and `yarn test` to ensure there are no linting or testing errors
- Verify the List component in Storybook

# Fixes
Fixes #50
